infra/create-app/provider.tf

```diff
@@ -3,6 +3,12 @@ provider "aws" {
   profile = var.profile
 }
 
+provider "aws" {
+  alias   = "useast1"
```

Comment:
Think this could be named better. I got confused in the file above thinking you were specifying the region rather than the alias. I can't think of a decent name though..

Reply:
I'm not super happy with it either, I was getting a few errors saying that "the provider the resource was created with has a different name" or some such when I tried to change it before. But I'll see if I can get around that and come up with a better name

Reply:
Now pushed!